cmd/user-agent/.env.production

```diff
@@ -0,0 +1 @@
+VUE_APP_ROUTER_HOST=http://localhost:10093
```

Comment:
why we need .env.production

Reply:
we need a router URL in our vue.js code, to access this variable we need a config file.

Reply:
agree, it doesn't help